packages/sdk/src/types/index.ts

```diff
@@ -25,7 +25,7 @@ type categoricalOutcomeIndex = [number, number];
 
 type scalarOutcomeIndex = [number, "Long" | "Short"];
 
-export type AssetId =
+export type AssetShortform =
```

Comment:
I don't think this type needs to exist, as it's an incomplete implementation of `Asset` and doesn't not contain enough information to create a proper `CategoricalOutcome` or `ScalarOutcome`.